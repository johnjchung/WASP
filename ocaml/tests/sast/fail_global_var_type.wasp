int a = 4;

Endpoint e(GET:) {
    float b = a;
}
