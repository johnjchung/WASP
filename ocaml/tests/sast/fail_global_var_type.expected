Fatal error: exception Type_checking.VariableDeclarationTypeError("Expected float expression, found int")
